knime_extension/src/nodes/spatialmodels.py

```diff
@@ -16,6 +16,22 @@
 __NODE_ICON_PATH = "icons/icon/SpatialModel/"
 
 
+@knext.parameter_group(label="ID Setting")
+class IDSetting:
```

Comment:
I wouldn't use a class for this single setting since it is also reflected in the node dialog and this way it looks like as if the other settings after the id belong to the ID Setting category e.g. Dependent variable in the 2SLS node.